broadinstitute/firecloud-orchestration
# CORE-165: even more passthroughs

Removes all calls to `passthrough` and replaces them with more generic `streamingPassthrough` calls. Removes the `passthrough` directive itself.

✅ passed unit tests

✅ passed swat tests

✅ manually tested while running locally:
1. used swagger-ui to check most of the APIs affected by this PR

✅ manually tested in a BEE:
1. create and share billing project
2. create workspace
3. import WDL from Dockstore to workspace
4. launch and complete a Cromwell submission


---

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Squash and merge. Make sure your branch deletes; GitHub should do this for you.
